### ISSUE TITLE
MD029/ol-prefix - Ordered list item prefix not detected in sublist

### ISSUE THREAD

**Issue by ogmios-voice** [DavidAnson/markdownlint#480]
In the following example instead of MD029/ol-prefix MD037/no-space-in-emphasis is reported (for every other line):

```
# text

* text
    2. text
        * text
        * text
        * text
        * text
```

**Comment by DavidAnson** [comment]
(The list items are indented more than necessary, but that is not the issue.)

CommonMark does not allow an ordered list to interrupt a paragraph unless it begins with the number 1. There is an example here in the specification and the explanation is just above: https://spec.commonmark.org/0.30/#example-304

Your example is therefore interpreted as a single unordered list item with asterisk characters embedded and so the errors you see reported are legitimate: https://dlaa.me/markdownlint/#%25m%23%20Issue%20480%0A%0A*%20text%0A%20%20%20%202.%20text%0A%20%20%20%20%20%20%20%20*%20text%0A%20%20%20%20%20%20%20%20*%20text%0A%20%20%20%20%20%20%20%20*%20text%0A%20%20%20%20%20%20%20%20*%20text%0A

Changing the 2 to a 1 in your example renders as expected and reports no errors.

**Comment by ogmios-voice** [comment]
Yes, changing 2 to 1 fixes the issue, but the reported errors basically do not state anything about this.
Also is there a way to continue an ordered list from a given number (>1)? (e.g. using some html tags / comments)

**Comment by DavidAnson** [comment]
I agree it would be nice if the tool could figure out all of the ways that Markdown might not do what the author intended, but in this case viewing the rendered output is a good clue about what's going wrong with the list definition.

For your question, please have a look at the discussion in #340 regarding starting a list at a different number.
